src/Ng/Helpers/EmptySecretInjector.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+using NuGet.Services.KeyVault;
+
+namespace Ng.Helpers
+{
+    public class EmptySecretInjector : ISecretInjector
```

Comment:
Why is this necessary?

Reply:
This job has not been converted yet to use the configuration JSON files to be used by the secret reader.